packages/requisitions/src/RequestRequisition/api/hooks.ts

```diff
@@ -144,3 +137,9 @@ export const useIsRequestRequisitionDisabled = (): boolean => {
     status === RequisitionNodeStatus.Sent
   );
 };
+
+export const useSaveRequestLines = () => {
+  const { store } = useHostContext();
```

Comment:
can now do
```
const { storeId } = useAuthContext();
```

Reply:
I've added the use of auth context .. I think we have the store in local storage, auth context, host context and a cooke, now? 😁